src/python/Python-2.7.10/setup.py

```diff
@@ -805,6 +805,9 @@ def detect_modules(self):
                                libraries=math_libs) )
         # Detect SSL support for the socket module (via _ssl)
         search_for_ssl_incs_in = [
+                              # On MacOS OpenSSL headers are installed using
+                              # Homebrew. Make sure they are picked first.
+                              '/usr/local/opt/openssl/include',
                               '/usr/local/ssl/include',
                               '/usr/sfw/include/',
```

Comment:
Sure… I know `/usr/sfw/include` is needed in Solaris 10. And the other two I have just verified, we have inherited them from upstream Python.